qa/suites/rbd/nvmeof/centos_latest.yaml

```diff
@@ -0,0 +1 @@
+.qa/distros/supported/centos_latest.yaml
```

Comment:
AFAIR, one of the requirements was to run the test on centos9/rhel9, @caroav please correct me if I'm wrong.
is this env request added somewhere for the nvmeof test execution?

Reply:
@barakda `centos_latest.yaml` is a symlink to `centos 9.stream.yaml` so it executes the test in centos 9.stream env only
if you want, I can use explicit `qa/distros/all/centos_9.stream.yaml` instead of `qa/distros/supported/centos_latest.yaml`?

Reply:
> @barakda `centos_latest.yaml` is a symlink to `centos 9.stream.yaml` so it executes the test in centos 9.stream env only if you want, I can use explicit `qa/distros/all/centos_9.stream.yaml` instead of `qa/distros/supported/centos_latest.yaml`?

Let's not do that.  It causes unnecessary churn when migrating to a new version of the distro.

Reply:
@VallariAg let's change to centos9.

Reply:
ok let's follow @idryomov comment and not change.